migration-guides/v8.0.0.md

```diff
@@ -0,0 +1,29 @@
+# v8.0.0 Migration Guide
```

Comment:
Do you think 2 and 3 belong in a migration guide or just this version's Release Notes? I'd like to keep this guide as slim as possible to singularly answer the question: "how do I upgrade my existing app?".

The details about what was changed and why can be left to the release notes.

Reply:
That sounds good. I'm not sure we have a good place to articulate release notes. I've moved this information into this PR's description. I also did this in the case of the lp-subsection-generator because it didn't have a migration guide. A lot of libraries seem to be getting rid of these migration guides and adding this information into the release definition in GitHub (what you mean by "release notes" I assume) anyway. 

Maybe we should update the PR template to include a Release Notes section, unless you have a better way of obtaining that information.

Reply:
Every release in GitHub has the opportunity to create release notes. E.g., https://github.com/LaunchPadLab/lp-components/releases/tag/v7.0.0. It's a little annoying that it isn't pulled from somewhere in `.github` for it to be version controlled and included in the review of a PR.

I'd be open to including an optional section in the PR template for Release Notes with a reminder to copy and paste them into the actual release once it's created.

Reply:
Added optional "Release Notes" section to the pull request template.